src/psyclone/transformations.py

```diff
@@ -1251,6 +1253,7 @@ def apply(self, target_nodes, options=None):
 
         return schedule, keep
 
+
 class OMPSingleTrans(ParallelRegionTrans):
     '''
     Create an OpenMP SINGLE region by inserting directives. The most
```

Comment:
Annoyingly, GitHub isn't letting me continue the conversations from the first review but the answer to:

> Which ones? I'm assuming ParseError and/or GenerationError since I don't check for them in the code fragment?

is "yes".